src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
I know that in readme nothing says about PropTypes, but I think there are needed